ruby/game/hangman_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'hangman'
+

```

Comment:
i think this rspec is coming along fine for now.  just work on my suggestions in the class and reach out if you have more questions